engine-core/src/main/java/io/nosqlbench/engine/core/lifecycle/ScenarioController.java

```diff
@@ -383,9 +383,8 @@ public void forceStopScenario(int waitTimeMillis, boolean rethrow) {
      * @param waitTimeMillis The time to wait, usually set very high
      * @return true, if all activities completed before the timer expired, false otherwise
      */
-    public boolean awaitCompletion(int waitTimeMillis) {
+    public boolean awaitCompletion(long waitTimeMillis) {
```

Comment:
this goes into `Thread.join(long millis)` and it was called with a compile time constant that overflowed the `int` type